### ISSUE TITLE
Implement proper filtering

### ISSUE THREAD

**Issue by recrsn** [agathver/ec2pricing#1]
Filtering currently is not bound to dropdown menus.

**Comment by Asutosh989** [comment]
Hey, @agathver I would like to work on it. Can you specify some more details as to what type of filtering you need?

**Comment by recrsn** [comment]
If you have successfully setup your local dev environment you can see that the contents of the dropdown in instance type and size do not correspond to each other.

That is if I select gpuCurrentGen, only g2.* instances should be available in the sizes dropdown.

**Comment by Asutosh989** [comment]
@agathver yes I could see that while running, can you specify from where the data is being fetched. After that I will start kicking on it.
